include/rangers-api/Application/UI/AppUITextInterface.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+namespace app::ui {
+    class AppUITextInterface : public hh::ui::UITextInterface {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.